Add brief Tobira "documentation"

owi92 opened the pull request:
This is a short text that explains what Tobira is (something along the lines of *"it's a video portal for your opencast."* in a few more words).

It is mainly meant to point out the existence of Tobira and includes a link to its own documentation for an in-depth guide on setup and configuration.

Part of https://github.com/opencast/opencast/issues/6727
<!--
Explain what this pull request does.  If this fixes a bug, also explain how to reproduce the issue.
-->

<!--
If your pull request is targeting the legacy (second oldest release branch), explain why this patch is so important that
it needs to go into the legacy and can not go just into stable or develop.
-->

### ~~How to test this patch~~

<!--
Explain how to test this patch. If this requires a particular set-up or configuration, explain that as well and provide
the necessary configuration if possible. The easier it is for others to test the patch, the faster this will get
reviewed and merged.
-->


### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)
* [ ] explain why it needs to be merged into the legacy branch, if it is targeting the legacy branch

mtneug reviewed (changes requested):
You pushed the same file twice with different spellings (`Tobira.md` vs `tobira.md`)

mtneug commented:
Open question from my side:

* Should this be merged after Tobira moved to the Opencast org?
* What is the plan with the Tobira documentation? Will Opencast and Tobira docs somehow merge? Should they use the same documentation system etc.?

owi92 committed:
Add brief Tobira documentation

This is mainly meant to point out the existence
of Tobira. Includes a link to its own documentation
for an in-depth guide on setup and configuration.

gregorydlogan commented on docs/guides/admin/docs/configuration/tobira.md:
```diff
@@ -0,0 +1,20 @@
+
+Tobira's main goal is to serve as a video portal for end users.
+**It is not meant to replace the Admin interface.**
+That should still be the preferred address for **admins** to manage publications.
```
This could be cleaned up a bit to read more like this

```suggestion
**It is not meant to replace the Admin interface, which is still be the preferred address for admins to manage publications.**
```

I'm still not happy with it, but it cleans up the `That`, which was somewhat unclear.

gregorydlogan commented on docs/guides/admin/docs/configuration/tobira.md:
```diff
@@ -0,0 +1,20 @@
+Tobira runs separately from Opencast and needs its own installation. The configuration necessary to connect these systems is mainly done in Tobira itself, but there are some things that you'll need to adjust in Opencast as well. For an in-depth explanation and the complete setup and configuration guide, see Tobira's main [documentation](https://elan-ev.github.io/tobira/setup).
+
+## Tobira in the Admin UI
+When you have connected a Tobira instance to your Opencast, the Admin UI offers some additional features: When creating a new series, you can choose in which Tobira page it should be mounted. That path can be also be changed or initialized later on.
```
```suggestion
When you have connected a Tobira instance to your Opencast, the Admin UI's series modal will now allow you to choose the series' mount point in Tobira.
```

If there are docs on what a `mount point` is, I would link to those.

owi92 replied:
Unfortunately, those don't really exist aside from some doc comments. I added a short explanation in brackets.

gregorydlogan reviewed (changes requested):
I would clean up these two spots, otherwise looks good
